src/science/mathematics/statistics/central-tendency.md

```diff
@@ -0,0 +1,88 @@
+### Measures of central tendency
+
+[![std-badge]][std]
+
+Measures of central tendency summarize a set of data as an "average" or "expected" value that is likely to be representative of a typical member of the data set in some way. The most common measures of central tendency are the
```

Comment:
The description should be shorter than this.  We don't need to know the implementation details, instead it is better to know from a high level what is going on, and linking API documentation.  Shoot for no more than 5 sentences / 2 paragraphs.  For such a simple task (calculating the mean) there is excessive description.

I would move the description of Option from Standard Deviation to mean's description.

Reply:
:+1: